Source/DafnyStandardLibraries/src/Std/Collections/Seq.dfy

```diff
@@ -774,6 +774,25 @@ module Std.Collections.Seq {
     }
   }
 
+  lemma {:induction false} LemmaMapPartialFunctionDistributesOverConcat<T,R>(f: (T --> R), xs: seq<T>, ys: seq<T>)
```

Comment:
I found this was necessary because MapPartialFunction was introduced, not sure if there's a better solution?